dmoj/executors/python_executor.py

```diff
@@ -53,7 +53,10 @@ def parse_feedback_from_stderr(self, stderr, process):
         if not match:
             return ''
         exception = match[0].group(1)
-        return '' if len(exception) > 20 else exception
+        if exception in __builtins__.__dict__ and issubclass(__builtins__.__dict__[exception], BaseException):
```

Comment:
`__builtins__` could either be a module or a `dict` depending on the execution environment. Maybe you should do `builtins = getattr(__builtins__, '__dict__', __builtins__)`.